data/human/human missions.txt

```diff
@@ -5733,3 +5733,228 @@ mission "Argosy Hijacking (Declined)"
+				to display
+					"Blind Man from Martini: taxi questions count" == 2
+				goto taxi
+
+			`	The taxi enters the University district proper now, and the window shows thongs of energetic university students from the University of New Sydney travelling to and from lectures and classrooms; going over steel walkways that run from building to building above you, and on footpaths beside the streets; clutching their backpacks and data pads. Your datapad shows that University isn't actually the only university district in New Sydney. The University of Chiron's campus is on the other side of town, presumably to keep the fierce competition between the two institutions from boiling over each other.`
+				to display
+					"Blind Man from Martini: taxi questions count" == 3
+				goto taxi
+
+			label hospital
+			`	Eventually, the taxi passes by a sign for the "UNS Health Sciences" quarter, which by the looks of it takes up an entire section of the University district. The taxi pulls up beside the entrance of the university hospital, where you help Douglas out with his luggage. The building is stark from the outside, to match the architecture of the surroundings, yet strangely colorful on the inside, as if the person building it decided that hospital interiors were far too unimaginative.`
+			`	After a short queue, you reach the reception with Douglas. The receptionist, placed behind a sterile-looking screen that looks far out of place with the colorful hallways, asks with a well-practiced smile, "How may I help you?"`
+			choice
+				`	"We're looking for Anita Pears. Where can we find her?"`
+			`	For a moment, the receptionist shuffles information around on her terminal. "Are one of you a parent?"`
+			`	"I'm the father, Douglas Pears," he replies.`
+			`	She takes a few more seconds before looking at you and saying, "Ah, but we cannot let you in. Only Mr. Pears is booked in for his daughter's procedure; we have a family-only policy, to keep the privacy of our patients."`
+			choice
+				`	"Alright."`
+					goto end
+				`	"Are you sure?"`
+			`	"We can't allow immediate exceptions, sorry. There's nothing I can do."`
+			label end
+			`	Douglas gives you a compassionate look, and says to the receptionist, "There'll be no worry." He turns back to you once more and says, "I'm sure that when Anita is recovered and well, we can meet again some day. I don't want to keep you from your business too long, either. I expect ship captains are busy folk." He comes up to you and, to your surprise, gives you a hug. "Thank you so much for bringing me here and coming along with me. I really can't say how I want to express my gratitude. But I hope we meet again. Farewell."`
```

Comment:
Maybe a reunion mission / thank you letter could be added.

Reply:
Yeah, I had thought about writing more - especially in the hospital - but I didn't because the mission was already pretty heavy, if you know what I mean, and there would be a lot of similarity to the Riley mission. It wouldn't be too much work to add a "thank you" dialogue, I guess.